app/renderer/controls/Button.tsx

```diff
@@ -19,12 +19,22 @@ const StyledButton = styled.span<StyledButtonProps>`
   background-color: ${(p) => p.theme.buttonPrimaryBg};
   color: ${(p) => p.theme.buttonPrimaryText};
   border-radius: 10px;
+  border: none;
   text-align: center;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
   font-size: 0.9em;
 
+  &:disabled {
```

Comment:
Instead of this I think you just need to add `p.disabled` to line 41 like so:

```css
${(p) => 
    !p.isLoading &&
    !p.disabled &&
    css`
```

This will take care of the pointer and background-color styling that is needed for when the button is disabled. And the prop on line 9 can be removed since `isDisabled` is not being passed to `StyledButton` anymore.